src/main/java/com/google/api/gax/grpc/OperationCallSettings.java

```diff
@@ -0,0 +1,116 @@
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.google.api.gax.grpc;
+
+import com.google.longrunning.Operation;
+import com.google.longrunning.OperationsApi;
+import com.google.protobuf.Message;
+import io.grpc.Channel;
+import io.grpc.MethodDescriptor;
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * A settings class to configure an OperationCallable for calls to a long-running API method
+ * (i.e. that returns the {@link Operation} type.)
+ */
+public final class OperationCallSettings<RequestT, ResponseT extends Message> {
+
+  private final SimpleCallSettings<RequestT, Operation> initialCallSettings;
+  private final Class<ResponseT> responseClass;
+
+  public final SimpleCallSettings<RequestT, Operation> getInitialCallSettings() {
+    return initialCallSettings;
+  }
+
+  protected OperationCallable<RequestT, ResponseT> createOperationCallable(
+      Channel channel, ScheduledExecutorService executor, OperationsApi operationsApi) {
+    UnaryCallable<RequestT, Operation> initialCallable =
+        initialCallSettings.create(channel, executor);
+    OperationCallable<RequestT, ResponseT> operationCallable =
+        new OperationCallable<>(
+            initialCallable, channel, executor, operationsApi, responseClass, this);
+    return operationCallable;
+  }
+
+  private OperationCallSettings(
+      SimpleCallSettings<RequestT, Operation> initialCallSettings, Class<ResponseT> responseClass) {
+    this.initialCallSettings = initialCallSettings;
+    this.responseClass = responseClass;
+  }
+
+  /**
+   * Create a new builder which can construct an instance of OperationCallSettings.
+   */
+  public static <RequestT, ResponseT extends Message> Builder<RequestT, ResponseT> newBuilder(
+      MethodDescriptor<RequestT, Operation> grpcMethodDescriptor, Class<ResponseT> responseClass) {
+    return new Builder<>(grpcMethodDescriptor, responseClass);
+  }
+
+  public final Builder<RequestT, ResponseT> toBuilder() {
+    return new Builder<>(this);
+  }
+
+  public static class Builder<RequestT, ResponseT extends Message> {
+    private SimpleCallSettings.Builder<RequestT, Operation> initialCallSettings;
+    private Class<ResponseT> responseClass;
+
+    public Builder(
+        MethodDescriptor<RequestT, Operation> grpcMethodDescriptor,
+        Class<ResponseT> responseClass) {
+      this.initialCallSettings = SimpleCallSettings.newBuilder(grpcMethodDescriptor);
+      this.responseClass = responseClass;
+    }
+
+    public Builder(OperationCallSettings<RequestT, ResponseT> settings) {
+      this.initialCallSettings = settings.initialCallSettings.toBuilder();
+      this.responseClass = settings.responseClass;
+    }
+
+    /**
+     * Set the call settings which are used on the call to initiate the operation.
+     */
+    public Builder setInitialCallSettings(
```

Comment:
That's because the response class is something that is "baked in" to a given callable; it works the same way as the `grpcMethodDescriptor`, which is also only provided at construction time. These are not something that users should ever programmatically.